src/styles/blocks/card.scss

```diff
@@ -0,0 +1,67 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__logo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &_info {
+    font-weight: 500;
+    font-size: 12px;
+    margin: 0 0 4px;
+    line-height: 18px;
+    color: $text-main;
+  }
+
+  &_kod {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-card;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review-col {
+    font-weight: 400;
+    font-size: 10px;
+    color: $text-main;
+    line-height: 14px;
+  }
+
+  &_buy-submit {
+    display: flex;
+    justify-content: center;
+    box-sizing: border-box;
+    font-size: 14px;
+    font-weight: 700;
+    color: #fff;
+    text-decoration: none;
+    height: 40px;
+    width: 166px;
+    background-color: #00acdc;
+    border-radius: 5px;
+    border: 1px solid #00acdc;
+    padding-top: 11px;
+    line-height: 16px;
+  }
+
+  &_buy-submit:hover {
+    color: #00acdc;
+    background-color: #fff;
+  }
```

Comment:
Check the 9th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)